python/classify.py

```diff
@@ -129,6 +148,27 @@ def main(argv):
     predictions = classifier.predict(inputs, not args.center_only)
     print("Done in %.2f s." % (time.time() - start))
 
+    if args.print_results:
+        with open(args.labels_file) as f:
+          labels_df = pd.DataFrame([
+               {
+                   'synset_id': l.strip().split(' ')[0],
+                   'name': ' '.join(l.strip().split(' ')[1:]).split(',')[0]
+               }
+               for l in f.readlines()
+            ])
+        labels = labels_df.sort('synset_id')['name'].values
```

Comment:
deprecated use of `sort`, now changed to `sort_values`.
~
http://pandas.pydata.org/pandas-docs/stable/generated/pandas.DataFrame.sort_values.html